### PR TITLE
update changelog v0.5

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v0.5.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Changelog since v0.4.0
+All notable changes to this project will be documented in this file.
+
```
Comment (qiujian16) [inline CHANGELOG/CHANGELOG-v0.5.md:3]: thanks, would you also add the contributor to each item?

Comment (zhiweiyin318) [reply CHANGELOG/CHANGELOG-v0.5.md:3]: done.

**Pull request merged**